src/Tool/ConfigDumperCommand.php

```diff
@@ -114,7 +124,7 @@ public function __invoke(array $args)
 
     /**
      * @param array $args
-     * @return stdClass
+     * @return object
```

Comment:
We don't know more about this? :thinking:

Reply:
Tried a union of help/error/command and psalm just hated it 🤷‍♂️